### PR TITLE
DONT PUSH UNTIL NOV 1: adds new items to gigabit portfolio

### LINKED ISSUE #2527: update Gigabit portfolio with 2018 projects

### PR DESCRIPTION
**Pull request by hannahkane**
Fixes #2527

**Changes proposed in this pull request:**
- adds new items to Gigabit Portfolio: https://learning.mozilla.org/en-US/gigabit/portfolio
- adds new cities to locations.json

### CODE REVIEW

**Comment by hannahkane** [comment]
@gvn - Gigabit team has requested that we don't push this live until November 1. Could you review in the meantime, please?

File: pages/gigabit/portfolio.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
...
     "Outcomes": "The GigLab has also been a boon to Chattanooga’s developer community – many other Mozilla Gigabit Fund projects demoed during November’s grand opening – as well as the Open Chattanooga Brigade, who have been been able to utilize the available bandwidth for processing large data sets. Planned future applications include pairing virtual reality headsets with city lidar and GIS data to build immersive educational experiences for younger patrons (a Minecraft-based project being piloted in conjunction with the Kansas City Digital Media Lab) and regular low-latency performances with musicians in other gigabit cities, including Burlington, VE.",
-    "Gigabit Advantage": "The technology publicly accessible in this space leverages Chattanooga's high-speed, low-latency network.",
+    "Gigabit Advantage": "The technology publicly accesssaible in this space leverages Chattanooga's high-speed, low-latency network.",
```
Comment (gvn) [inline pages/gigabit/portfolio.json]: Spelling?

**Comment by hannahkane** [comment]
Um...took me two tries, but I think I got the spelling right this time.

@gvn - are you around after Mozfest to push this? Could be Thursday, the 2nd, since you'll be out on the 1st.

**Comment by gvn** [comment]
Yeah, I can push on the 2nd when I get back. Anyone with access to Heroku should be able to push as well.

**Pull request merged**